.github/workflows/buildcli.yml

```diff
@@ -0,0 +1,148 @@
+# This pipeline is used as template to build the sfpowerscripts plugin
+
+
+name: 'sfpowerscripts Plugin Build'
+
+on:
+  workflow_call:
+    inputs:
+      commitToGit:
```

Comment:
commitToGit, stageToDeploy, environmentToDeploy, organization, buildNumber, toBuild, and bump parameters can be removed, as they're not being used/referenced